Javascript/q44.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q44</title>
+
+  <style>
+    body {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    th {
+      text-align: left;
+    }
+  </style>
+
+  <script>
+    function myFunction() {
+      const length = document.querySelector("#demo0").value;
+      const count = document.querySelector("#demo1").value;
```

Comment:
Please cast these variables as `Number()`. Currently you are using these values as strings. JS kinda sorta manages to handle it some times but trying to use a string as a number is wrong

Reply:
Yeah, you will still get the the `value` as a string rather than a `number`. That `type="number"` attribute only tells the browser to only accept number values from the user in that input. But it does not mean that it will give you the value as a `number` though